zingolib/src/lightclient/test_features.rs

```diff
@@ -4,22 +4,53 @@
 //! raw_receiver.0:   A &str representing the receiver address
 //! raw_receiver.1:   A u64 representing the number of zats to be sent
 //! raw_receiver.2:   An Option<&str> that contains memo data if not None
-use nonempty::NonEmpty;
 use zcash_client_backend::{
     zip321::{TransactionRequest, Zip321Error},
     PoolType, ShieldedProtocol,
 };
-use zcash_primitives::transaction::TxId;
 
 use crate::{
-    data::{proposal::TransferProposal, receivers::transaction_request_from_receivers},
+    data::receivers::transaction_request_from_receivers,
     error::ZingoLibError,
-    utils::conversion::{address_from_str, testing::receivers_from_send_inputs},
+    utils::conversion::{address_from_str, zatoshis_from_u64},
     wallet::Pool,
 };
 
-use super::{propose::ProposeSendError, send::send_with_proposal::QuickSendError, LightClient};
+use crate::lightclient::LightClient;
 
+use zingoconfig::ChainType;
+
+use crate::data::receivers::Receivers;
+
+/// Converts primitive rust types to zcash receiver types for constructing transaction requests.
+///
+/// # Panics
+///
+/// Panics if the address, amount or memo conversion fails.
+pub fn receivers_from_send_inputs(
```

Comment:
the reason why I created a conversion module is to home functions like this that should not belong in lightclient because they are not a method of lightclient. this doesnt seem right to me maybe we shuld discuss

Reply:
I agree that it is an odd duck.   I am OK with it moving more.   I just moved it because I knew it was only in test.   In a subsequent PR I moved it into zingo-testutils..   I am not clear on where it will ultimately live.   I am a fan of putting where it's used.